specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview/securityStandards.json

```diff
@@ -0,0 +1,394 @@
+          },
+          "Create or update security standard over management group scope": {
+            "$ref": "./examples/SecurityStandards/PutByManagementGroupSecurityStandard_example.json"
+          }
+        },
+        "tags": [
+          "SecurityStandards"
+        ],
+        "description": "Creates or updates a security standard over a given scope",
+        "operationId": "SecurityStandards_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/StandardId"
+          },
+          {
+            "$ref": "#/parameters/SecurityStandardBody"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/SecurityStandard"
+            }
+          },
+          "200": {
+            "description": "OK - Updated",
+            "schema": {
+              "$ref": "#/definitions/SecurityStandard"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../common/v2/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete a security standard over subscription scope": {
+            "$ref": "./examples/SecurityStandards/DeleteBySubscriptionSecurityStandard_example.json"
+          },
+          "Delete a security standard over security connector scope": {
+            "$ref": "./examples/SecurityStandards/DeleteBySecurityConnectorSecurityStandard_example.json"
+          },
+          "Delete a security standard over management group scope": {
+            "$ref": "./examples/SecurityStandards/DeleteByManagementGroupSecurityStandard_example.json"
+          }
+        },
+        "tags": [
+          "SecurityStandards"
+        ],
+        "description": "Delete a security standard over a given scope",
+        "operationId": "SecurityStandards_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/Scope"
+          },
+          {
+            "$ref": "#/parameters/StandardId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - security standard was deleted"
+          },
+          "204": {
+            "description": "No Content - security standard does not exist"
+          },
+          "default": {
+            "description": "Error response that describes why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/types.json#/definitions/CloudError"
```

Comment:
>"../../../common/v2/types.json#/definitions/CloudError" [](http://example.com/codeflow?start=22&length=55)

please always consider use common type in
../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse